lib/tasks/npm-task.js

```diff
@@ -28,6 +29,10 @@ class NpmTask extends Task {
     this.versionConstraints = '3 || 4 || 5 || 6';
   }
 
+  get packageManagerOutputName() {
+    return this.packageManager.name === 'yarn' ? 'Yarn' : this.packageManager.name;
```

Comment:
Seems odd to capitalize here, why can't we just use this:

```suggestion
    return this.packageManager.name;
```

?

Reply:
Yarn capitalizes the name on their website 🤷‍♂️

Reply:
K, let's go with my suggestion (don't accept it though because you'll have to make other changes too).